goat/placeholder.py

```diff
@@ -20,3 +20,20 @@ def multiply_by_2(x: int) -> int:
     if type(x) == int:
         return x * 2
     raise TypeError("The type of the input wasn't int.")
+    
+def add_two(x):
```

Comment:
I think you should add typehints to the function.

Reply:
Typehint is added.